eng/packages/http-client-csharp/emitter/src/sdk-context-options.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { CreateSdkContextOptions } from "@azure-tools/typespec-client-generator-core";
+
+export const azureSDKContextOptions: CreateSdkContextOptions = {
+    versioning: {},
+    additionalDecorators: [
+        "Azure\\.ClientGenerator\\.Core\\.@useSystemTextJsonConverter",
```

Comment:
How is this used?

Reply:
this is not in use for now, add it in advance.
The usage should like in autorest.csharp https://github.com/Azure/autorest.csharp/blob/0834c7fc1dba137246db481a1d24d58dbabd63e5/src/AutoRest.CSharp/Common/Input/InputTypes/InputModelType.cs#L43-L54